core/src/main/java/io/questdb/network/Net.java

```diff
@@ -103,10 +107,24 @@ public static void dump(long buffer, int len) {
         }
     }
 
+    public static native long freeAddrInfo(long pAddrInfo);
+
     public static native void freeMsgHeaders(long msgHeaders);
 
     public native static void freeSockAddr(long sockaddr);
 
+    public static long getAddrInfo(LPSZ host, int port) {
+        return getAddrInfo(host.address(), port);
+    }
+
+    public static long getAddrInfo(CharSequence host, int port) {
+        try (Path p = new Path().of(host).$()) {
```

Comment:
this feels a bit abusive. It's not a filesystem Path. 

I think a natural candidate would be `CharSequenceZ`, but unfortunately, it supports ASCII only. Is utf8 support the reason why you picked `Path`?

Reply:
utf8 is a can of worms I would prefer not to open here

![image](https://user-images.githubusercontent.com/7276403/174645783-b5989849-e5bc-41e1-a64c-90f1a172634b.png)